Intermittent failure in p2p_segwit.py "AssertionError: No exception raised"

https://travis-ci.org/github/bitcoin/bitcoin/jobs/738951476#L3128:
```
2020-10-26T12:45:49.099000Z TestFramework (ERROR): Assertion failed
Traceback (most recent call last):
  File "/home/travis/build/bitcoin/bitcoin/ci/scratch/build/bitcoin-s390x-linux-gnu/test/functional/test_framework/test_framework.py", line 125, in main
    self.run_test()
  File "/home/travis/build/bitcoin/bitcoin/ci/scratch/build/bitcoin-s390x-linux-gnu/test/functional/p2p_segwit.py", line 308, in run_test
    self.test_superfluous_witness()
  File "/home/travis/build/bitcoin/bitcoin/ci/scratch/build/bitcoin-s390x-linux-gnu/test/functional/p2p_segwit.py", line 319, in func_wrapper
    func(self, *args, **kwargs)
  File "/home/travis/build/bitcoin/bitcoin/ci/scratch/build/bitcoin-s390x-linux-gnu/test/functional/p2p_segwit.py", line 2107, in test_superfluous_witness
    assert_raises_rpc_error(-22, "TX decode failed", self.nodes[0].decoderawtransaction, serialize_with_bogus_witness(tx).hex())
  File "/home/travis/build/bitcoin/bitcoin/ci/scratch/build/bitcoin-s390x-linux-gnu/test/functional/test_framework/util.py", line 123, in assert_raises_rpc_error
    assert try_rpc(code, message, fun, *args, **kwds), "No exception raised"
AssertionError: No exception raised
```

--- a/test/functional/p2p_segwit.py
+++ b/test/functional/p2p_segwit.py
@@ -2097,14 +2097,14 @@ class SegWitTest(BitcoinTestFramework):
 
         raw = self.nodes[0].createrawtransaction([{"txid": unspent['txid'], "vout": unspent['vout']}], {self.nodes[0].getnewaddress(): 1})
         tx = FromHex(CTransaction(), raw)
-        assert_raises_rpc_error(-22, "TX decode failed", self.nodes[0].decoderawtransaction, serialize_with_bogus_witness(tx).hex())
+        assert_raises_rpc_error(-22, "TX decode failed", self.nodes[0].decoderawtransaction, hexstring=serialize_with_bogus_witness(tx).hex(), iswitness=True)
         with self.nodes[0].assert_debug_log(['Superfluous witness record']):
             self.test_node.send_and_ping(msg_bogus_tx(tx))
         raw = self.nodes[0].signrawtransactionwithwallet(raw)
         assert raw['complete']
         raw = raw['hex']
         tx = FromHex(CTransaction(), raw)
-        assert_raises_rpc_error(-22, "TX decode failed", self.nodes[0].decoderawtransaction, serialize_with_bogus_witness(tx).hex())
+        assert_raises_rpc_error(-22, "TX decode failed", self.nodes[0].decoderawtransaction, hexstring=serialize_with_bogus_witness(tx).hex(), iswitness=True)
         with self.nodes[0].assert_debug_log(['Unknown transaction optional data']):
             self.test_node.send_and_ping(msg_bogus_tx(tx))
 

